tslint.json

```diff
@@ -13,10 +13,6 @@
     "curly": true,
     "eofline": true,
     "forin": true,
-    "import-blacklist": [
-      true,
-      "rxjs"
```

Comment:
Since RxJS v6 we can remove this.

Reply:
thank you for the info 👍  I will remove that too in other several [packages](https://github.com/angular-material-extensions)